src/lib/utility/utilityWindows.cpp

```diff
@@ -1,4 +1,5 @@
 #include "utilityWindows.h"
+#include <cassert>
```

Comment:
Do we need this include here? On my VS2017 it compiles without.

Reply:
I have no idea how this got into PR, deleted this line